From f79d1d1bf9e6f54b67e5482602084fbff7fd9cc9 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Fri, 28 Oct 2016 12:50:34 -0300
Subject: [PATCH] redis.conf: adjust defauts for buildroot

Based on Martin Bark's patch hence archlinux package.
We just need to specify a proper db directory.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 redis.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/redis.conf b/redis.conf
index 22e00bc..3bb0430 100644
--- a/redis.conf
+++ b/redis.conf
@@ -244,7 +244,7 @@ dbfilename dump.rdb
 # The Append Only File will also be created inside this directory.
 #
 # Note that you must specify a directory here, not a file name.
-dir ./
+dir /var/lib/redis/
 
 ################################# REPLICATION #################################
 
-- 
2.7.3

